builtin/logical/aws/path_config_root.go

```diff
@@ -23,6 +23,14 @@ func pathConfigRoot() *framework.Path {
 				Type:        framework.TypeString,
 				Description: "Region for API calls.",
 			},
+			"iamendpoint": &framework.FieldSchema{
```

Comment:
The auth backend already has iam_endpoint and sts_endpoint; it'd be good to underscore-separate this here.